com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.Debug.cs

```diff
@@ -30,6 +30,13 @@ public partial class HDRenderPipeline
 
         DebugDisplaySettings m_DebugDisplaySettings = new DebugDisplaySettings();
 
+        public static bool allowDebugMaterialSamplerOverride = UnityEngine.SystemInfo.hasMaterialSamplerOverride;
```

Comment:
@sebastienlagarde do you know if this is also available in URP? should we let them know we added this feature since they are redoing also the debug menu (@arttu-peltonen is helping out actually)

Reply:
Shouldnt this boolean be moved to DebugDisplaySettings > DebugData ?

Reply:
> @sebastienlagarde do you know if this is also available in URP? should we let them know we added this feature since they are redoing also the debug menu (@arttu-peltonen is helping out actually)

There's no mip bias overrides in URP (nor was it available in builtin from what I can tell). Perhaps it's a useful feature to add later on? I think our most immediate focus is to achieve URP<->builtin feature parity.

Reply:
The feature (mip bias) is in builtin, so available for any pipeline.
Not sure that there is really a need to display it in URP (neither in HDRP as it is only here to check that the feature works, but otherwise in practice I don't really see a usage for it - maybe check mips of texture?) 
Anyway, it could be cross platform yeah if we really want.

Reply:
https://ono.unity3d.com/unity/unity/pull-request/122268/_/graphics/mipbias

Reply:
i think Andre was suggesting it could be nice to add it to URP too - see https://unity.slack.com/archives/G077VATKP/p1615452059011200?thread_ts=1615398479.007700&cid=G077VATKP 
@sebastienlagarde should we add a jira for this?

Reply:
Dont' be confuse between the feature itself which works for all pipeline and this debug tool which is mainly to check if implementation works. Andre refer to having the feature.
It is very little work, so we can add it easily if needed, it is just that I don't see any usage in debug menu, neither for HDRP outside of validating it works :).

Reply:
ok so lets make sure we have the feature working for URP too and not only let UX know that URP interface will not support this for now.

Reply:
* I'll move the boolean to the debug settings.  Thanks for the feedback @jenniferd-unity . Keep it coming 👍 
* This can be easily added to the URP debug settings. I actually tried to, but I quickly discovered there was no debug menu. I did tested this on a user script, and works in URP as well. It will be up to you guys if you guys want it there once that debug menu is up and running. Should be easy to add.

Reply:
Yeah URP debug menu is still in a branch and hasn't landed. If somebody needs it later in URP, feel free to direct them to me and I can add it to the debug menu.

Reply:
Sounds good. We are doing now discussions on the API, it hasnt been finalized :) But once its official and in the trunk ill ping you.